pkg/internal/checkup/executor/testpmd/console.go

```diff
@@ -93,13 +93,13 @@ func (t TestpmdConsole) Run() error {
 
 	testpmdCmd := buildTestpmdCmd(t.vmiEastNICPCIAddress, t.vmiWestNICPCIAddress, t.vmiEastEthPeerMACAddress, t.vmiWestEthPeerMACAddress)
 
-	resp, err := console.SafeExpectBatchWithResponse(t.vmiSerialClient, t.namespace, t.vmiName,
-		[]expect.Batcher{
-			&expect.BSnd{S: testpmdCmd + "\n"},
-			&expect.BExp{R: testpmdPrompt},
-			&expect.BSnd{S: "start" + "\n"},
-			&expect.BExp{R: testpmdPrompt},
-		},
+	testpmdConsole := console.NewExpecter(t.vmiSerialClient, t.namespace, t.vmiName)
```

Comment:
We already have a type called `TestpmdConsole`, and a variable with the same name.
Could you please rename this variable?

Reply:
changed to `testpmdConsoleExpecter`